guide/src/telemetry.md

```diff
@@ -65,127 +66,133 @@ see Prometheus-encoded metrics.
+queries{chain="ibc-1",query_type="query_consensus_state"} 143
+queries{chain="ibc-1",query_type="query_consensus_states"} 1
 queries{chain="ibc-1",query_type="query_latest_height"} 1
-queries{chain="ibc-1",query_type="query_packet_acknowledgements"} 3
-queries{chain="ibc-1",query_type="query_packet_commitments"} 6
+queries{chain="ibc-1",query_type="query_packet_acknowledgements"} 1
+queries{chain="ibc-1",query_type="query_packet_commitments"} 4
 queries{chain="ibc-1",query_type="query_staking_params"} 2
-queries{chain="ibc-1",query_type="query_txs"} 190
-queries{chain="ibc-1",query_type="query_unreceived_packets"} 15
-queries{chain="ibc-2",query_type="query_application_status"} 33
-queries{chain="ibc-2",query_type="query_blocks"} 1
-queries{chain="ibc-2",query_type="query_channel"} 12
-queries{chain="ibc-2",query_type="query_client_connections"} 2
-queries{chain="ibc-2",query_type="query_client_state"} 24
-queries{chain="ibc-2",query_type="query_clients"} 1
-queries{chain="ibc-2",query_type="query_connection"} 2
-queries{chain="ibc-2",query_type="query_connection_channels"} 2
-queries{chain="ibc-2",query_type="query_consensus_state"} 29
-queries{chain="ibc-2",query_type="query_consensus_states"} 2
-queries{chain="ibc-2",query_type="query_latest_height"} 1
-queries{chain="ibc-2",query_type="query_packet_acknowledgements"} 3
-queries{chain="ibc-2",query_type="query_packet_commitments"} 6
-queries{chain="ibc-2",query_type="query_staking_params"} 2
-queries{chain="ibc-2",query_type="query_txs"} 192
-queries{chain="ibc-2",query_type="query_unreceived_packets"} 11
-# HELP send_packet_count Number of SendPacket relayed.
+queries{chain="ibc-1",query_type="query_txs"} 32
+queries{chain="ibc-1",query_type="query_unreceived_acknowledgements"} 71
+# HELP send_packet_count Number of SendPacket events processed
 # TYPE send_packet_count counter
-send_packet_count{chain="ibc-0",channel="channel-0",counterparty="ibc-2",port="transfer"} 5
-send_packet_count{chain="ibc-1",channel="channel-0",counterparty="ibc-0",port="transfer"} 6
-send_packet_count{chain="ibc-2",channel="channel-1",counterparty="ibc-1",port="transfer"} 7
+send_packet_count{chain="ibc-1",channel="channel-0",counterparty="ibc-0",port="transfer"} 35
+# HELP tx_latency_confirmed The latency for all transactions submitted & confirmed to a specific chain, i.e. the difference between the moment when Hermes received a batch of events until the corresponding transaction(s) were confirmed. Milliseconds.
+# TYPE tx_latency_confirmed histogram
+tx_latency_confirmed_bucket{chain="ibc-0",channel="channel-0",counterparty="ibc-1",port="transfer",le="10"} 0
+tx_latency_confirmed_bucket{chain="ibc-0",channel="channel-0",counterparty="ibc-1",port="transfer",le="50"} 0
+tx_latency_confirmed_bucket{chain="ibc-0",channel="channel-0",counterparty="ibc-1",port="transfer",le="100"} 0
+tx_latency_confirmed_bucket{chain="ibc-0",channel="channel-0",counterparty="ibc-1",port="transfer",le="200"} 0
```

Comment:
Can we update the metrics here with some nicer numbers?

Reply:
I ran some commands in order to have better telemetry output example for the Hermes guide